spec/sample_artifacts/version_12.tap

```diff
@@ -0,0 +1,1456 @@
+ok 532 A pair of servers can establish a join in a v2 room
+ok 533 Outbound federation rejects send_join responses with no m.room.create event
+ok 534 Outbound federation rejects m.room.create events with an unknown room version
+ok 535 Outbound federation can send events
+ok 536 Inbound federation can receive events
+ok 537 Inbound federation can receive redacted events
+ok 538 Inbound federation can return events
+#** TODO: Unhandled incoming event of type 'm.room.message' at lib/SyTest/Federation/Server.pm line 462.
+not ok 539 Inbound federation redacts events from erased users
+# Started: 2019-07-31 19:44:49.878
+# Ended: 2019-07-31 19:44:50.239
+# HTTP Request failed ( 500 Internal Server Error https://localhost:8800/_matrix/client/r0/account/deactivate?access_token=MDAxY2xvY2F0aW9uIGxvY2FsaG9zdDo4ODAwCjAwMTNpZGVudGlmaWVyIGtleQowMDEwY2lkIGdlbiA9IDEKMDAzYmNpZCB1c2VyX2lkID0gQGFub24tMjAxOTA3MzFfMTk0MTA1LTYzNDpsb2NhbGhvc3Q6ODgwMAowMDE2Y2lkIHR5cGUgPSBhY2Nlc3MKMDAyMWNpZCBub25jZSA9IHR6Xl9zZG5WNnRrVE1Ha3gKMDAyZnNpZ25hdHVyZSDKgHrR9JLV7lVljzvG5FNkBtSIAr50fV9Vg37S6bZ3dwo ) from POST https://localhost:8800/_matrix/client/r0/account/deactivate?...
+# {"errcode":"M_UNKNOWN","error":"Internal server error"}
+# 0.023827: Registered new user @anon-20190731_194105-634:localhost:8800
+# 0.347576: Fetched event before erasure
+# {
+#   origin => "localhost:8800",
+#   origin_server_ts => JSON::number(1564602290221),
+#   pdus => [
+#     {
+#       auth_events      => [
+#                             [
+#                               "\$1564602289347MAinE:localhost:8800",
+#                               { sha256 => "SCNucT2CJRLagBkAOI1YUNAst94ujxmV2RftNv91lg0" },
+#                             ],
+#                             [
+#                               "\$1564602289348objZO:localhost:8800",
+#                               { sha256 => "2oWoKVFbUhetRRUILtSMjrnB97zCnBqqI4jG5A4sq60" },
+#                             ],
+#                             [
+#                               "\$1564602289349OJayc:localhost:8800",
+#                               { sha256 => "Jkc/SCSd7ABUpTkKmkX2gtXSrNtrvwWaudfCv4ZneOc" },
+#                             ],
+#                           ],
+#       content          => { body => "body1", msgtype => "m.text" },
+#       depth            => JSON::number(7),
+#       event_id         => "\$1564602290353cAuGt:localhost:8800",
+#       hashes           => { sha256 => "UOE4n9e+eDH3jEpZtSmFohpVQxCM4X72jZtJvKGVcjw" },
+#       origin           => "localhost:8800",
+#       origin_server_ts => JSON::number(1564602290180),
+#       prev_events      => [
+#                             [
+#                               "\$18:localhost:37429",
+#                               { sha256 => "ikAE36M1b+/ILQT+ZiBX/mHjfu+cqirNFzd5zpc+iyc" },
+#                             ],
+#                           ],
+#       prev_state       => [],
+#       room_id          => "!wZvvAsAETTvjVBBjcY:localhost:8800",
+#       sender   
```

Comment:
I would add a bit of yaml to this test file
```
not ok 3 Resolve address
 ---
 message: "Failed with error 'hostname peebles.example.com not found'"
 severity: fail
 data:
   got:
     hostname: 'peebles.example.com'
     address: ~
   expected:
     hostname: 'peebles.example.com'
     address: '85.193.201.85'
 ...
```
I took that directly from https://testanything.org/tap-version-13-specification.html

```
[1] pry(#<RSpec::ExampleGroups::TestSummaryBuildkitePluginInput::Tap::Version12>)> puts input.failures[14].details
message: "Failed with error 'hostname peebles.example.com not found'"
severity: fail
data:
  got:
    hostname: 'peebles.example.com'
    address: ~
  expected:
    hostname: 'peebles.example.com'
    address: '85.193.201.85'
=> nil
```

Reply:
or...hang on I guess yaml is not part of the 12 spec?